### PR TITLE
Add TitleId TextView in App Dialog

### PR DESCRIPTION
**Pull request by dima-xd**
Also if you hold with touch the Title ID it will copy it to clipboard

### CODE REVIEW

**Comment by AbandonedCart** [comment]
Out of curiosity, what is the purpose of having it copy to the clipboard?

**Comment by dima-xd** [comment]
> 

For example when you need to know Title ID of the game to import save file

**Comment by AbandonedCart** [comment]
Oh ok. That makes sense. It is a pain to look them up.

**Comment by AndresNorm** [comment]
Well, this is great.
Less hassle for googling the ID now

File: app/src/main/java/emu/skyline/AppDialog.kt [hunk 1]
```diff
@@ -91,6 +92,12 @@ class AppDialog : BottomSheetDialogFragment() {
             shortcutManager.requestPinShortcut(info.build(), null)
         }
 
+        binding.gameTitleId.setOnLongClickListener {
+            val clipboard = requireActivity().getSystemService(android.content.Context.CLIPBOARD_SERVICE) as android.content.ClipboardManager
+            clipboard.setPrimaryClip(android.content.ClipData.newPlainText("Title ID", item.titleId))
```
Comment (lynxnb) [inline app/src/main/java/emu/skyline/AppDialog.kt:97]: Show a snack bar after you do this saying "Copied to clipboard".

**Review by lynxnb — CHANGES_REQUESTED** [review]

File: app/src/main/java/emu/skyline/AppDialog.kt (hunk 1 above)
Comment (dima-xd) [reply app/src/main/java/emu/skyline/AppDialog.kt:98]: > Show a snack bar after you do this saying "Copied to clipboard".

Done. Also added this line to `strings.xml`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -194,6 +194,7 @@
     <!-- Software Keyboard -->
     <string name="input_hint">Input Text</string>
     <!-- Misc -->
+    <string name="copied_to_clipboard">Copied to clipboard</string>
```
Comment (lynxnb) [inline app/src/main/res/values/strings.xml:197]: Don't make a new Misc section for this, but move this at the top of the file, in the Common section.

**Review by lynxnb — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dima-xd) [reply app/src/main/res/values/strings.xml:197]: > Don't make a new Misc section for this, but move this at the top of the file, in the Common section.

It's not new btw, it was just an empty section

Comment (lynxnb) [reply app/src/main/res/values/strings.xml:197]: Right, in that case delete it, and move to Common.

Comment (dima-xd) [reply app/src/main/res/values/strings.xml]: Done

**Review by lynxnb — APPROVED** [review]

**Pull request merged**